cypress/support/page_objects/sliderMove.js

```diff
@@ -0,0 +1,36 @@
+export class SliderMove {
+
+    movingSlider() {
+
+        const currentValueMin = 0;
+        const targetValueMin = 3;
+        const currentValueMax = 5;
+        const targetValueMax = 3;
+        const increment = 1;
+        const setTargetValueMin = (targetValueMin - currentValueMin) / increment;
+        const setTargetValueMax = (currentValueMax - targetValueMax) / increment;
+        const rightArrow = '{rightarrow}'.repeat(setTargetValueMin);
+        const leftArrow = '{leftarrow}'.repeat(setTargetValueMax);
+        
+        
+        cy.get('[data-test="slider"] .ngx-slider-pointer-min')
+            .should('have.attr', 'aria-valuetext', '0')
+            .type(rightArrow)
+
+        cy.get('[data-test="slider"] .ngx-slider-pointer-min')
```

Comment:
This part of code is hard to understand for someone who is not familiar with cypress/css/this particular slider. But if we would move this code to separate method, for example something like this:

shouldHaveValueOf/hasValue(value) {
    cy.get('[data-test="slider"] .ngx-slider-pointer-max')
            .should('have.attr', 'aria-valuetext', value)
}

then cypress/css/slider would be just an implementation detail, I would just read the method name and know what it does. Also you could reuse this piece of code and be sure it always work the same way.

Reply:
I'll try to correct but I have to think how to separate these methods because we don't move only left and right, because we have two points to move on left and on right